### PR TITLE
Improve kernel structure to be the main repo from CC

### PR DESCRIPTION
**Pull request by miguelinux**
To set packaging as main repo some improvements were made.

- Now, update_kernel.sh fetch kernel version from upstream
- Keeps a release track version
- Deleted files not needed files in repo
- Updates CC mailing list address

### CODE REVIEW

File: kernel/update_kernel.sh [hunk 1]
```diff
@@ -6,26 +6,48 @@
 # Default image to build is the one specified in file configure.ac
 # located at the root of the repository.
 set -x
+KR_URL=https://www.kernel.org/releases.json
+KR_LTS=4.9
+
 AUTHOR=${AUTHOR:-$(git config user.name)}
 AUTHOR_EMAIL=${AUTHOR_EMAIL:-$(git config user.email)}
 
-CC_VERSIONS_FILE="../versions.txt"
```
Comment (gorozco1) [inline kernel/update_kernel.sh:12]: why ignoring ../versions.txt?

File: kernel/update_kernel.sh [hunk 2]
```diff
@@ -6,26 +6,48 @@
...
-source "$CC_VERSIONS_FILE"
-clear_vm_kernel_version=$(echo $clear_vm_kernel_version | cut -d'-' -f1)
-VERSION=${1:-$clear_vm_kernel_version}
```
Comment (gorozco1) [inline kernel/update_kernel.sh:15]: we need to be able to build "any" version of the kernel with parameter

**Review by gorozco1 — CHANGES_REQUESTED** [review]
please clarify

File: kernel/update_kernel.sh (hunk 1 above)
Comment (miguelinux) [reply kernel/update_kernel.sh:12]: The idea is that packaging/kernel will be the first place where the kernel will be updated and after build test and a PR to packaging  will be propagated to others repos

File: kernel/update_kernel.sh (hunk 2 above)
Comment (miguelinux) [reply kernel/update_kernel.sh:15]: Ok, I'll fix it

**Comment by miguelinux** [comment]
@gorozco1 changes are integrated

File: kernel/update_kernel.sh [hunk 3]
```diff
@@ -2,30 +2,62 @@
...
-echo "Update linux-container to: $VERSION-$next_release"
+get_kernel ()
+{
+    KSOURCE=$1
+    KFILE=${KSOURCE##*/}
+
+    if [ ! -f ${KFILE} ]
+    then
+        echo "Downloading linux kernel from ${KSOURCE}"
+        if ! curl --fail -O ${KSOURCE}
```
Comment (gorozco1) [inline kernel/update_kernel.sh]: Please remove the conditional for "file" and always try the download of the kernel with the continue flag.


`curl -L -O -C - "${KSOURCE}"`

**Comment by miguelinux** [comment]
@gorozco1 the change is done, Could you review? please.

File: kernel/update_kernel.sh [hunk 4]
```diff
@@ -96,4 +76,36 @@ then
...
+echo "Running: $0 $@"
+echo "Update linux-container to: $VERSION-$next_release"
+
+changelog_update $VERSION
+
+sed "s/\@VERSION\@/$VERSION/g; s/\@RELEASE\@/$next_release/g" linux-container.spec-template > linux-container.spec
+
+ln -sf linux-$VERSION.tar.xz linux-container_$VERSION.orig.tar.xz
+
+for i in `cat obs_working_dir/debian/patches/series`
```
Comment (MarioCarrilloA) [inline kernel/update_kernel.sh]: Maybe you could use $() instead of ``

**Comment by MarioCarrilloA** [comment]
@miguelinux could you please update README file too :)

**Comment by miguelinux** [comment]
@gorozco1, @MarioCarrilloA changes done

**Review by gorozco1 — APPROVED** [review]

**Pull request merged**